op-node/rollup/derive/channel.go

```diff
@@ -146,25 +144,21 @@ func (ch *Channel) Reader() io.Reader {
 
 // BatchReader provides a function that iteratively consumes batches from the reader.
 // The L1Inclusion block is also provided at creation time.
-func BatchReader(cfg *rollup.Config, r io.Reader, l1InclusionBlock eth.L1BlockRef) (func() (BatchWithL1InclusionBlock, error), error) {
```

Comment:
Why do we no longer track the L1 Inclusion block?

Reply:
In the original code, `BatchReader()` returns `BatchWithL1InclusionBlock` but the L1 inclusion block is not used in `ChannelInReader`

`ChannelInReader` [just picks `BatchData`](https://github.com/ethereum-optimism/optimism/blob/develop/op-node/rollup/derive/channel_in_reader.go#L83-L92) from the `BatchWithL1InclusionBlock`, and [the L1 inclusion block is set again in `BatchQueue`.](https://github.com/ethereum-optimism/optimism/blob/develop/op-node/rollup/derive/batch_queue.go#L134-L137)

So, this change is to remove unnecessary code.